### PR TITLE
feat: loadBalancerSourceRanges in chaos-dashboard service

### LINKED ISSUE #3920: Support for LB in dashboard svc
## Feature Request

**Describe the feature you'd like:**
For dashboard k8s service we don't have support to add loadbalancer source IPs.

`{{- if and .Values.dashboard.service.type (eq .Values.dashboard.service.type "LoadBalancer") }}
  {{- with .Values.dashboard.service.loadBalancerSourceRanges }}
  loadBalancerSourceRanges:
  {{- toYaml . | nindent 4 }}
 {{- end }}
{{- end }}`

### PR DESCRIPTION
**Pull request by jamshidi799**
## What problem does this PR solve?
This PR adds support of loadBalancerSourceRanges to chaos-dashboard service in the helm chart

 Close #3920 

## Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`

## Cherry-pick to release branches (optional)

> This PR should be cherry-picked to the following release branches:

- [ ] release-2.6
- [ ] release-2.5

### CHANGELOG

> Must include at least one of them.

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

### Tests

> Must include at least one of them.

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test

### Side effects

- [ ] **Breaking backward compatibility**

## DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml [hunk 1]
```diff
@@ -193,3 +193,10 @@ spec:
       targetPort: {{ .Values.dashboard.env.METRIC_PORT }}
       name: metric
 {{- end }}
```
Comment (g1eny0ung) [inline helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml:195]: The below changes should be inserted before this `{{- end }}`.

File: helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml [hunk 2]
```diff
@@ -193,3 +193,10 @@ spec:
...
+
+{{- if and .Values.dashboard.service.type (eq .Values.dashboard.service.type "LoadBalancer") }}
```
Comment (g1eny0ung) [inline helm/chaos-mesh/templates/chaos-dashboard-deployment.yaml:197]:
```suggestion
{{- if and .Values.dashboard.service.type (eq .Values.dashboard.service.type "LoadBalancer") .Values.dashboard.service.loadBalancerSourceRanges }}
```

**Review by g1eny0ung — CHANGES_REQUESTED** [review]

**Comment by jamshidi799** [comment]
now it should be ok

**Review by g1eny0ung — APPROVED** [review]
LGTM!

**Comment by jamshidi799** [comment]
thank you @g1eny0ung

**Pull request merged**